src/axom/mint/tests/mint_execution_cell_traversals.cpp

```diff
@@ -127,21 +135,28 @@ void check_for_all_cells_ijk( )
 
   // STEP 0: create the test mesh
   using MESH = typename internal::mesh_type< MeshType >::MeshType;
-  MESH* test_mesh = dynamic_cast< MESH* >( internal::create_mesh< MeshType >( uniform_mesh ) );
+  MESH* test_mesh =
+    dynamic_cast< MESH* >( internal::create_mesh< MeshType >( uniform_mesh ) );
   EXPECT_TRUE( test_mesh != nullptr );
 
-  IndexType* icoords = test_mesh->template createField< IndexType >( "i", CELL_CENTERED );
-  IndexType* jcoords = test_mesh->template createField< IndexType >( "j", CELL_CENTERED );
-  IndexType* kcoords = test_mesh->template createField< IndexType >( "k", CELL_CENTERED );
+  IndexType* icoords = test_mesh->template createField< IndexType >( "i",
+                                                                     CELL_CENTERED );
+  IndexType* jcoords = test_mesh->template createField< IndexType >( "j",
+                                                                     CELL_CENTERED );
+  IndexType* kcoords = test_mesh->template createField< IndexType >( "k",
+                                                                     CELL_CENTERED );
 
   for_all_cells< ExecPolicy, xargs::ijk >( test_mesh,
```

Comment:
@corbett5 , can you please restrict the ``uncrustify`` changes to Sidre and Core which are related to this PR and not any other file changes in Mint. Unfortunately, ``uncrustify`` doesn't deal well with the formatting of Lambdas and until I have time to tweak what that looks like, I prefer we don't apply formatting to Mint. I'll deal with this in another PR.

Reply:
Done.